tools/dptools/README

```diff
@@ -71,14 +71,14 @@ You can install the script package via the standard 'python setup'
 mechanism. If you want to install it system-wide into your normal
 python installation, you simply issue::
 
-  python setup.py
+  python3 setup.py
```

Comment:
Shouldn't we recommend installation via `pip install .` instead?

Reply:
Probably in addition, yes.
But... there is a namespace collision:
https://pypi.org/project/dptools/
so instructions will need to be done carefully.